src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Storage<T> {
+    public static final List<User> USERS_STORAGE = new ArrayList<>();
+    public static final List<Bet> BETS_STORAGE = new ArrayList<>();
```

Comment:
these are not constants despite the modifiers are the same
let's use lower case

Reply:
done